<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prop</class>
 <widget class="QMainWindow" name="prop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ÈÅìÂÖ∑Ê†è</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>771</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Papyrus&quot;;</string>
     </property>
     <property name="text">
      <string>Ââ©‰ΩôÈáëÂ∏ÅÊï∞Ôºö</string>
     </property>
    </widget>
    <widget class="QWidget" name="centralwidget_3" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>271</width>
       <height>751</height>
      </rect>
     </property>
     <widget class="QPushButton" name="SetDiceNum">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>È¢ÑËÆæÈ™∞Â≠êÁÇπÊï∞üé≤</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Wave">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>ÂÜ≤ÂáªÊ≥¢üí•</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SpeedUp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>Âä†ÈÄüÂô®üÜô</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SlowDown">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>ÂáèÈÄüÂô®ü§ö</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MoveBackward">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>ÂèçÂêëÁßªÂä®üîô</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Babysit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>È¢úËâ≤ÊâòÁÆ°üôá</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SwapLoca">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>‰∫§Êç¢‰ΩçÁΩÆüîÑ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Explosion">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>590</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>Ëá™ÁàÜ‚ò†Ô∏è</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetDiceTime">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>670</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 235, 237);</string>
      </property>
      <property name="text">
       <string>ÊîπÂèòÊâîÈ™∞Â≠êÊ¨°Êï∞üé∞</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SetDN_spinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>73</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="SetDT_spinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>716</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="SetDN_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Wave_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Speed_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Slow_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Back_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>370</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Baby_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>450</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Swap_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>530</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Explosion_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>610</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="SetDT_Tleft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>690</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buyDN">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buySU">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buySD">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buyBW">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buyBS">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>440</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buySWAP">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>520</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buyDT">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>680</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>600</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 233, 220);</string>
      </property>
      <property name="text">
       <string>Ë¥≠‰π∞</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>     ÈÅìÂÖ∑</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>-10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> Ââ©‰ΩôÊ¨°Êï∞</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Rank" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Death</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Yellow</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Green_k">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Green_d">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Yellow_k">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Yellow_d">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Blue_k">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Red_k">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Blue_d">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Red_d">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ddadwd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="cheatButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 233, 220);</string>
     </property>
     <property name="text">
      <string>Ê∞™Èáë</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="money_left">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Pay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="color_now">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ÂΩìÂâçÁé©ÂÆ∂Ôºö</string>
     </property>
    </widget>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>3</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
